src/main/java/uk/gov/hmcts/reform/cwrdapi/client/domain/CaseWorkerProfile.java

```diff
@@ -81,6 +81,10 @@ public class CaseWorkerProfile extends CaseWorkerDomain implements Serializable
     @JsonProperty("role")
     private List<Role> roles;
 
+    @MappingField(clazz = Skill.class)
```

Comment:
No need to add this. Can you please remove this?

Reply:
Needful done

Reply:
Thanks @prudhvi-maddineni